### PR TITLE
Replace code comments with debug var

### PR DESCRIPTION
**Pull request by kj7rrv**
It's easier to use now

### CODE REVIEW

**Comment by mr-martian** [comment]
personally I don't mind having a bit of commented out debugging code

**Comment by kj7rrv** [comment]
Okay. Maybe have a debug variable?

**Comment by kj7rrv** [comment]
'if debug: print(variable)' instead of '#print(variable)'

**Comment by mr-martian** [comment]
If you want

**Comment by kj7rrv** [comment]
@mr-martian how does this look?

File: malaphor/malaphor.py [hunk 1]
```diff
@@ -17,11 +19,15 @@
 else:
 	input_idiom = sys.stdin.read()
 
+if debug:
+    n = 1 #Number of words it will replace
```
Comment (khannatanmai) [inline malaphor/malaphor.py]: Never really used, it was more of a future idea to have a control on the number of words it will replace

Comment (kj7rrv) [reply malaphor/malaphor.py]: Okay.

**Comment by kj7rrv** [comment]
See #6

**Pull request merged**